src/forms/datepicker/DatePicker.tsx

```diff
@@ -0,0 +1,423 @@
+                                                                                            ? true
+                                                                                            : false
+                                                                                        : isToday
+                                                                                        ? true
+                                                                                        : false
+                                                                                }
+                                                                                onClick={this.handleSelectedDate.bind(
+                                                                                    this,
+                                                                                    calendar.toDate(),
+                                                                                )}
+                                                                            >
+                                                                                {calendar.date()}
+                                                                            </button>
+                                                                        </div>
+                                                                    </td>
+                                                                );
+                                                                calendar.add(1, "day");
+                                                                return day;
+                                                            })}
+                                                        </tr>
+                                                    );
+                                                })}
+                                            </tbody>
+                                        </table>
+                                    </div>
+                                )}
+                                {viewMode === ViewMode.Month && (
+                                    <div className="monthpicker">
+                                        {[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11].map((month: number) => {
+                                            return (
+                                                <button
+                                                    type="button" //prettier
+                                                    className={classNames([
+                                                        "calendar-btn", //prettier
+                                                        "month",
+                                                        month === 0 && "is-selected",
+                                                    ])}
+                                                    autoFocus={month === 0 ? true : false}
```

Comment:
can be replaced on !month. But it doesn't work if you need number only

Reply:
Yeah, we need to keep as number.